### PR TITLE
[enterprise-4.13] OCPBUGS#13406: Update the ansible and helm base image version tags

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.13 only

Issue: https://issues.redhat.com/browse/OCPBUGS-13406

Link to docs preview: https://60072--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-updating-projects.html#osdk-upgrading-projects_osdk-helm-updating-projects

https://60072--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/ansible/osdk-ansible-updating-projects.html#osdk-upgrading-projects_osdk-ansible-updating-projects

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by kelbrown20 — COMMENTED** [review]
Hello, great work! Just some formating nits and things I noticed with the previews to look over.

File: modules/osdk-updating-v125-to-v128.adoc [hunk 1]
```diff
@@ -40,6 +40,16 @@ The following procedure updates an existing {type}-based Operator project for co
...
 .Procedure
 
+ifdef::helm[]
+. Find the `ose-helm-operator` pull spec in Dockerfile, and update the image tag to `v4.13`:
++
+[source,terminal]
+----
+FROM registry.redhat.io/openshift4/ose-helm-operator:v4.12 <1>
+----
+<1> Update the tag version from `v4.12` to `v4.13`.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/osdk-updating-v125-to-v128.adoc]: Fixed this problem using ifeval

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**